test/corpus/constants.txt

```diff
@@ -423,6 +423,37 @@ do
                   (decimal
                     (int)))))))))))
 
+================================================================================
+float
+================================================================================
+
+do
+  1.
+  1.1
+  1e+12
+  1e-12
+  1.2e+12
```

Comment:
Should probably also add tests for `1.f` and `1.0f`.

Reply:
The `float32` type is treated separately in the grammar. 
But I see we should probably add a test for the `1.0f` case https://github.com/ionide/tree-sitter-fsharp/blob/be2acafdf5fc78762f6958b1e25ca4e63ad6c96c/test/corpus/constants.txt#L291